Pod/Classes/Swift/VOKCoreDataManager+Swift.swift

```diff
@@ -12,13 +12,38 @@ Swiftier versions of some Vokoder functions that return more exact managed objec
 public extension VOKCoreDataManager {
     
     /**
+     A swiftly named wrapper for the singleton.
+     
+     - returns: The shared instance of the core data manager
+     */
+    @nonobjc
+    public static var shared: VOKCoreDataManager {
+        return self.sharedInstance()
+    }
+    
+    /**
+     DEPRECATED: use managedObject(ofClass:inContext:) instead
      Create an appropriately typed instance of a given NSManagedObject subclass.
      
      - parameter objectClass: The class of object to create
      - parameter context: The managed object context in which to create the object or nil for the main context (defaults to nil)
      - returns: A new instance of the given class
      */
-    public func managedObjectOfClass<T: NSManagedObject>(_ objectClass: T.Type,
+    @available(*, deprecated: 4.1.0, message: "use managedObject(ofClass:inContext:) instead")
+    public func managedObjectOfClass<T: NSManagedObject>(_ objectClass: T.Type, inContext context: NSManagedObjectContext? = nil) -> T {
+        // Force-unwrapped since the method below will fatal error if it's not there,
+        // and the compiler is too stupid to realize this is already typed.
+        return self.managedObject(of: objectClass, in: context) as! T
```

Comment:
This isn't calling the method below, it's calling the other method that the method below also calls. That might explain why the compiler isn't cooperating on the type.